docs/OnTrack/Deployment/Enhanced Authentication/current-and-proposed-authentication-evaluation-5.1.md

```diff
@@ -0,0 +1,185 @@
+# Authentication Evaluation – Review of Current and Proposed Authentication Solutions
+
+[Overview](#overview)
+
+[The State of the Current Authentication Mechanisms](#the-state-of-the-current-authentication-mechanisms)
+
+- [The Current Authentication Setup](#the-current-authentication-setup)
+- [Risks and Issues](#risks-and-issues)
+
+[Proposed Authentication Enhancements](#proposed-authentication-enhancements)
+
+- [The Proposed Authentication Mechanism](#the-proposed-authentication-mechanism)
+- [Advancements of the Previous Authentication Mechanisms and how it Addresses Issues of the Old Method](#advancements-of-the-previous-authentication-mechanisms-and-how-it-addresses-issues-of-the-old-method)
+- [Potential Issues and Concerns that must be Considered](#potential-issues-and-concerns-that-must-be-considered)
+
+## Overview
+
+The purpose of this documentation is to formally review the current authentication mechanisms which
+are in place within the OnTrack architecture and compare this to the proposed authentication
+solution to be developed by the Enhance Authentication team. Within this evaluation, the current
+authentication setup and mechanisms will be described, and then analysed in terms of the risks and
+issues encompassed within such a setup. The new authentication solution which is proposed to be
+developed and implemented will then be described, analysing how this new system aims to advance the
+authentication capabilities of its predecessor and how it will work to mitigate the risks and issues
+of the current system which it will replace. Additionally, the issues and concerns which may develop
+as a result of the implementation of the proposed authentication system will be examined, including
+the regulatory and compliance considerations which must be addressed during the development of the
+new system.
+
+## The State of the Current Authentication Mechanisms
+
+### The Current Authentication Setup
+
+Currently, the OnTrack system relies upon the authentication provided by its client, Deakin
+University. The authentication mechanism utilised by Deakin University is a Single Sign On (SSO)
+system which utilises Multi-Factor Authentication (MFA) technology called “Duo Security”. As part of
```

Comment:
Suggestion: Duo Security is a company and MFA is the technology that we utilise from them.

```suggestion
system which utilises Multi-Factor Authentication (MFA) technology through _Duo Security_. As part of
```